package-rfcx/rfcx/audio.py

```diff
@@ -101,7 +101,7 @@ def downloadStreamSegments(token, dest_path, stream_id, min_date, max_date, gain
         Args:
             token: RFCx client token.
             dest_path: Audio save path.
-            stream_id: RFCx guardian id
+            stream_id: RFCx stream id
```

Comment:
Try not to write descriptions that have zero information. If it is called `stream_id` then I know it is the stream id without your comment.

Suggest you only add descriptions if they add extra information.

`stream: Identifies a stream/site`

Also your `min_date` could explain how it modifies the behaviour of the function.

`min_date: Only include files starting after this timestamp (ISO format, e.g. 2021-01-20T01:02:30.000Z)`

👆I'm a developer and this makes my job easier when calling your API!